.changeset/curvy-dingos-jump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-azure-sites': minor
+---
+
+Show Azure site tags in EntityAzureSitesOverviewWidget
```

Comment:
```suggestion
'@backstage/plugin-azure-sites': patch
---

Show Azure site tags in `EntityAzureSitesOverviewWidget`.
```

Lowering to `patch` since this plugin is on `0.x` and there's no breaking change